ext_localconf.php

```diff
@@ -8,14 +8,24 @@
         // extension name, matching the PHP namespaces (but without the vendor)
         'Tea',
         // arbitrary, but unique plugin name (not visible in the BE)
-        'Tea',
+        'TeaIndex',
```

Comment:
We also do the same kind of changes in `Configuration/TCA/Overrides/tt_content.php` to make the plugin usable as a content element.

Reply:
That's true. I'm wondering if existing typoscript configuration (plugin.tx_tea) is enough to handle those plugin names